src/main/webapp/WEB-INF/themes/desktop/ftl/article/amendmentGroupTitle.ftl

```diff
@@ -0,0 +1,23 @@
+<#--
```

Comment:
The boilerplate you get when templates are variables, and variables are templates, and :copyright: